### ISSUE TITLE
Differentiate without color (2.0)

### ISSUE THREAD

**Issue by danispringer** [lichess-org/mobile#408]
Please ensure the app is usable with a grayscale filter, such that, for example, the green circle indicating opponent being online or not, and other functions, can be distinguished without the need for color (such as, for example, actual text "online/offline").

A good example of this is iOS microphone/camera usage indicators, which turn from circles to squares, so you know which is being shown, even if you cannot tell color.

And a bad example of this (or a great example of what not to do...), is WhatsApp's "blue checks", where the entire UI functionality is fully dependent on the assumption that a user's screen (and eyes) can tell the colors apart.

**Comment by veloce** [comment]
This is a good point and I should test the app in black and white. I think the green dot can be distinguished though, because it is plain when online and empty when offline.

**Comment by DaniSpringer** [comment]
@veloce i personally have a very hard time telling if my opponent is offline even with colors filter enabled. For whatever that’s worth. On app or website, light and dark mode.

**Comment by veloce** [comment]
So we need to replace the green dot by a different icon for online and offline statuses. That is fine by me, I just need to find theses icons.

**Comment by veloce** [comment]
https://api.flutter.dev/flutter/cupertino/CupertinoIcons/wifi-constant.html

and 

https://api.flutter.dev/flutter/cupertino/CupertinoIcons/wifi_slash-constant.html

look like good candidates.

**Comment by veloce** [comment]
@danispringer feel free to open other issues if you notice more color differentiation problems.

**Comment by danispringer** [comment]
@veloce thank you!
